pkgs/development/tools/parsing/nex/default.nix

```diff
@@ -0,0 +1,25 @@
+{ buildGoPackage, fetchFromGitHub, lib }:
+let
+  rev = "1a3320dab988372f8910ccc838a6a7a45c8980ff";
+in
+buildGoPackage rec {
```

Comment:
Please use buildGoModule

Reply:
upstream does not check in a `go.mod`; please advise

Reply:
Oh, yeah then we can't use it. Probably best to add a very short comment that the question does not come up again.